submissions/api.py

```diff
@@ -447,27 +447,27 @@ def get_all_course_submission_information(course_id, item_type, read_replica=Tru
     if read_replica:
         submission_qs = _use_read_replica(submission_qs)
 
-    query = submission_qs.select_related('student_item').prefetch_related('score_set').filter(
+    query = submission_qs.select_related('student_item__scoresummary__latest__submission').filter(
```

Comment:
My guess is that most people will only want the latest relevant score with no historical data for something like a download, so that should be the default behavior. But I think there are valid reasons for wanting every submission and its corresponding score or scores in the future.